src/days-2-4-adv/adv.py

```diff
@@ -49,3 +72,57 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+
+
+
+
+
+valid_directions = {"n": "n", "s": "s", "e": "e", "w": "w",
+                    "forward": "n", "backwards": "s", "right": "e", "left": "w"}
+
+player = Player(input("What is your name? "), room['outside'])
+
+#print(player.currentRoom)
+
+while True:
+    cmds = input("-> ").lower().split(" ")
+    if len(cmds) == 1:
+        if cmds[0] == "q":
+            break
+        elif cmds[0] in valid_directions:
+            player.travel(valid_directions[cmds[0]])
+        elif cmds[0] == "look":
+            player.look()
+        else:
+            print("I did not understand that cmds.")
+    else:
+
+        if cmds[0] == "look":
+            if cmds[1] in valid_directions:
+                player.look(valid_directions[cmds[1]])
+    else:
+        if cmds[0] == ('get' or 'take'):
```

Comment:
This else statement on line 104 is throwing an error - I think the one on 99 might need to be an elif? Because I don't think you can have two else statements for a conditional.